### DISCUSSION TITLE
What is joining the org about?

### DISCUSSION THREAD

**Discussion by maphew** [TriliumNext/Notes#36] [General]
Regarding #1, what does joining  the organization mean, do, enable, ...? What is above and beyond the github infrastructure? For example participating in these discussion threads, creating issues, contributing code via pull requests etc. are already possible.

I'm not a Trilium user at present, though I might become one if it's reasonable to expect it will continue to be an active project. I've read so many good things about it. If joining the org is a way to push towards vibrancy, I'm happy to add my voice. If joining the team is better suited to folks who are ready to pull weight _now_ it'd probably be better to cheer from the sidelines.  So, what is joining the org about? why join?

**Comment by meichthys** [comment]
Very good question. Ive wondered the same thing.

At this point everyone who has requested to join has been accepted. It really doesn't change much. Here's the official github docs about membership: https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-personal-account-on-github/managing-your-membership-in-organizations/about-organization-membership


Only organization owners can change member privileges which are defined here: https://docs.github.com/en/organizations/managing-peoples-access-to-your-organization-with-roles/roles-in-an-organization


There are currently 3 owners: https://github.com/orgs/TriliumNext/people?query=role%3Aowner

Although there has been great interest in new membership, (43 members so far), only a very small percent of those members have yet to contributeb perhaps due to no clear goal as of yet. But such is the nature of open source - participation is rare and organization is usually (and understandably) poor due to the fact that everyone's involvement is a hobby and ancillary to day jobs and other primary sources of income. Perhaps TriliumNext can be one of the few that bucks the trend and has enough interest and participation to keep the project truly open source and vibrant! Thanks for your interest in the project, and I suggest you join membership and participate where you can!

At some point in the future it may be wise to remove membership of members that have not contributed in any way, but at this point I think we are just trying to drum up as much interest and enthusiasm as possible!

**Comment by maphew** [comment]
Thank you for the details. From my reading everything I can conceive of
myself contributing can be done from the outside. A member will have to
review and accept them, but that'd be best anyway.

> There are currently 3 owners:
https://github.com/orgs/TriliumNext/people?query=role%3Aowner

I only see one, @Nriver.

**Reply by meichthys** [reply]
I'm not sure why, but when logged into github with a TriliumNext member account I see 3 owners.

**Reply by Nriver** [reply]
Because I'm the only one who made the visibility to public :)
I'm not the owner, just an ordinary member here.
